release/k3s/k3s.go

```diff
@@ -252,31 +255,31 @@ func getAuth(privateKey string) (ssh.AuthMethod, error) {
 	return publicKeys, nil
 }
 
-func (r *Release) gitRebaseOnto() error {
+func (r *Release) gitRebaseOnto() (string, error) {
 	dir := filepath.Join(r.Workspace, "kubernetes")
 
 	// clean kubernetes directory before rebase
 	if err := cleanGitRepo(dir); err != nil {
-		return err
+		return "", err
 	}
 	if _, err := runCommand(dir, "rm", "-rf", "_output"); err != nil {
-		return err
+		return "", err
 	}
 
 	commandArgs := strings.Split(fmt.Sprintf("rebase --onto %s %s %s-k3s1~1",
```

Comment:
Can we add another entry with `2>&1`?

Reply:
I tried it in a couple of different ways, but it always ends up with an error:
```
FATA[0005] failed to rebase and create tags: exit status 129: usage: git rebase [-i] [options] [--exec <cmd>] [--onto <newbase> | --keep-base] [<upstream> [<branch>]]
```